package.json

```diff
@@ -33,7 +33,7 @@
     "cross-env": "^5.1.4",
     "cz-conventional-changelog": "^2.1.0",
     "enzyme": "^3.3.0",
-    "enzyme-adapter-react-16": "^1.1.1",
+    "enzyme-adapter-react-16": "^1.5.0",
     "enzyme-context-patch": "^0.0.9",
```

Comment:
I had added the context patch before to work with the previous version of enzyme adapter react. But looks like the latest version includes it? In that case we can remove the patch again.

Reply:
This is what also is probably breaking the build since the context patch is trying to patch enzyme adapter